pkg/kubectl/resource/visitor.go

```diff
@@ -648,6 +648,16 @@ func RetrieveLazy(info *Info, err error) error {
 	return nil
 }
 
+// CreateAndRefresh creates an object from input info and refreshes info with that object
+func CreateAndRefresh(info *Info) error {
```

Comment:
Is this function original move from `pkg/kubectl/cmd/create.go` and made it public ? if so, should we delete the previous one ?